code/modules/admin/verbs/buildmode.dm

```diff
@@ -297,3 +313,42 @@
 				if(holder.throw_atom)
 					holder.throw_atom.throw_at(object, 10, 1)
 					log_admin("[key_name(usr)] threw [holder.throw_atom] at [object]",admin_key=key_name(usr))
+
+/verb/load_template_verb()
+	set name = "Load Template"
+	set category = "Special Verbs"
+	if(!usr)
+		return
+	load_template(usr)
+
+/proc/load_template(var/mob/user)
+	if(!user)
+		return
+
+	if(!check_rights(R_SPAWN))
+		return
+
+	var/list/templates
+	try
+		templates = json_decode(return_file_text("config/templates_list.json"))
+	catch(var/exception/ej)
+		log_debug("Warning: Could not load the templates config as templates_list.json is missing - [ej]")
+		return
+
+	if(!templates || !templates["templates_list"] || templates["templates_folder"] == "")
+		return
+
+	var/turf/T = get_turf(user)
+	var/name = input(user, "Which template would you like to load?", "Load Template", null) as null|anything in templates["templates_list"]
+	
+	if (!name || !T)
+		return
+
+	var/datum/map_template/maploader = new (templates["templates_folder"] + name, name)
+	if (!maploader)
+		log_debug("Error, unable to load maploader in proc load_template!")
+		return
+
+	var/centered = input(user, "Do you want template to load as center or Edge?", "Load Template", null) as null|anything in list("Center", "Edge")
+	maploader.load(T, centered == "Center" ? TRUE : FALSE)
+	log_and_message_admins("[key_name_admin(user)] has loaded template [name] at the coordinates [T.x], [T.y], [T.z].", user, T)
```

Comment:
psure the coords are already logged with `log_and_message_admins` when you pass `T`.

Reply:
Yeah, you are right. I forgot to remove them